### PR TITLE
Fix/ changed the style for the vertical arrow button

### PR DESCRIPTION
**Pull request by juliahermak**
Screenshot 2023-12-29 08 21 34
Screenshot 2023-12-29 08 21 26

### CODE REVIEW

**Comment by juliahermak** [comment]
[UI: Place the "vertical arrow" button for navigating to the end and beginning of the coin list inside a semi-transparent surrounding circle](https://trello.com/c/gSTkJMt7/463-ui-place-the-vertical-arrow-button-for-navigating-to-the-end-and-beginning-of-the-coin-list-inside-a-semi-transparent-surroundin)

**Review by bludnic — APPROVED** [review]

File: src/views/Home.vue [hunk 1]
```diff
@@ -275,8 +275,16 @@ export default {
       }
       :deep(.v-tabs .v-slide-group__prev .v-icon),
       :deep(.v-tabs .v-slide-group__next .v-icon) {
-        color: map-get($adm-colors, 'primary2');
-        pointer-events: none;
+        z-index: 1;
+        color: map-get($adm-colors, 'primary');
+        border-radius: 50%;
+        background-color: transparentize(map-get($adm-colors, 'primary2'), 0.1);
```
Comment (adamant-al) [inline src/views/Home.vue:281]:
```suggestion
        background-color: transparentize(map-get($adm-colors, 'primary2'), 0.3);
```

File: src/views/Home.vue [hunk 2]
```diff
@@ -304,8 +312,16 @@ export default {
...
```
Comment (adamant-al) [inline src/views/Home.vue:318]:
```suggestion
        background-color: transparentize(map-get($adm-colors, 'primary2'), 0.7);
```

**Review by adamant-al — APPROVED** [review]
Less opacity means more transparency.
Approve changes and merge.

**Pull request merged**